### PR TITLE
feat(cli): show absolute path for output path of current Prisma

### PR DESCRIPTION
**Pull request by mnmt7**
closes #7771 
/claim #7771

### CODE REVIEW

File: packages/internals/src/engine-commands/getEnginesMetaInfo.ts [hunk 1]
```diff
@@ -145,7 +145,16 @@ export async function resolveEngine(binaryName: BinaryType): Promise<EngineInfo>
   )()
 
   const engineInfo: EngineInfo = {
-    path: enginePathEither,
+    path: pipe(
+      enginePathEither,
+      E.fold(
+        (error) => E.left(error),
+        (path) => {
+          const absolutePath = `${process.cwd()}/${path}`
```
Comment (rishi-raj-jain) [inline packages/internals/src/engine-commands/getEnginesMetaInfo.ts:153]: you might want to do join here instead of forwardSlash as it'd break on Windows.

**Comment by mnmt7** [comment]
@rishi-raj-jain Thanks for pointing that out. I have now changed the code to use join instead of concatenating.

**Comment by jharrell** [comment]
@mnmt7 please fill out the CLA so that I can pass this along to engineering.

see: https://github.com/prisma/prisma/pull/20954#issuecomment-1705744203

**Comment by mnmt7** [comment]
> @mnmt7 please fill out the CLA so that I can pass this along to engineering.
> 
> see: [#20954 (comment)](https://github.com/prisma/prisma/pull/20954#issuecomment-1705744203)

@jharrell Done

**Comment by jharrell** [comment]
@mnmt7 I believe this approach is on the right path, but there are no tests confirming or denying that. Please write a test that fails on main and succeeds with your implementation. 

Thanks!

**Review by jharrell — CHANGES_REQUESTED** [review]
@mnmt7 I believe this approach is on the right path, but there are no tests confirming or denying that. Please write a test that fails on main and succeeds with your implementation. 

Thanks!

**Comment by mnmt7** [comment]
@jharrell While creating the test, I realized that I made the wrong change. The `resolveEngine()` function gives the absolute path of the engine and it was the `getEnginesInfo()` function that converted it into the relative path. Closing this in favor of #21115. Here I have added the test as well.

**Pull request closed without merging**